### ISSUE TITLE
Fatal error in launcher: Unable to create process using

### ISSUE THREAD

**Issue by ppetrucz** [bmaltais/kohya_ss#241]
I've encountered an error that prevents the lora training to start. I was able to train one lora, then I restarted the tool and the error appeared when I press the train button with the same configs I used for the successful build. I'm using the latest version just installed it today.

The error:

Folder 100_anna: 2200 steps
max_train_steps = 2200
stop_text_encoder_training = 0
lr_warmup_steps = 0
accelerate launch --num_cpu_threads_per_process=2 "train_network.py" --pretrained_model_name_or_path="runwayml/stable-diffusion-v1-5" --train_data_dir="D:/stable-diffusion/base images/anna style training/LORA/LORA/image" --resolution=512,512 --output_dir="D:/stable-diffusion/base images/anna style training/LORA/LORA/model" --logging_dir="D:/stable-diffusion/base images/anna style training/LORA/LORA/log" --network_alpha="128" --save_model_as=safetensors --network_module=networks.lora --text_encoder_lr=5e-5 --unet_lr=0.0001 --network_dim=128 --output_name="anna 22" --lr_scheduler_num_cycles="1" --learning_rate="0.0001" --lr_scheduler="constant" --train_batch_size="1" --max_train_steps="2200" --save_every_n_epochs="1" --mixed_precision="fp16" --save_precision="fp16" --seed="1234" --caption_extension=".txt" --cache_latents --optimizer_type="AdamW8bit" --max_data_loader_n_workers="1" --clip_skip=2 --bucket_reso_steps=64 --mem_eff_attn --gradient_checkpointing --xformers --bucket_no_upscale
Fatal error in launcher: Unable to create process using '"D:\kohya_ss\venv\Scripts\python.exe"  "D:\stable-diffusion\kohya_ss\venv\Scripts\accelerate.exe" launch --num_cpu_threads_per_process=2 "train_network.py" --pretrained_model_name_or_path="runwayml/stable-diffusion-v1-5" --train_data_dir="D:/stable-diffusion/base images/anna style training/LORA/LORA/image" --resolution=512,512 --output_dir="D:/stable-diffusion/base images/anna style training/LORA/LORA/model" --logging_dir="D:/stable-diffusion/base images/anna style training/LORA/LORA/log" --network_alpha="128" --save_model_as=safetensors --network_module=networks.lora --text_encoder_lr=5e-5 --unet_lr=0.0001 --network_dim=128 --output_name="anna 22" --lr_scheduler_num_cycles="1" --learning_rate="0.0001" --lr_scheduler="constant" --train_batch_size="1" --max_train_steps="2200" --save_every_n_epochs="1" --mixed_precision="fp16" --save_precision="fp16" --seed="1234" --caption_extension=".txt" --cache_latents --optimizer_type="AdamW8bit" --max_data_loader_n_workers="1" --clip_skip=

**Comment by ppetrucz** [comment]
Figured I’ve moved the tool’s folder to another location.

**Comment by Zhalorous** [comment]
I moved the folder too. Any idea how to fix this? Fatal error in launcher: Unable to create process using '"D:\kohya_ss\venv\Scripts\python.exe"  The bit above is referencing the old location for me.

**Comment by ppetrucz** [comment]
I simply moved it back to where it was and it worked.
If you really want to change its location you might need to completely reinstall it, but I’m not sure on that.

**Comment by Teriss** [comment]
If you move the folder, you need to change the python path written in the exe file in the venv/scripts.

1. 1.Just open all exe files with a text editor(such as notepad++), search for the python path inside and replace it with a relative path. For example, change "D:\Code\kohya_ss\venv\script\python.exe" to "python.exe"
2. If you are using a standalone python rather than a system environment, you need to add ". /python310/python.exe venv venv" (change it to your own path) to reset the virtual environment settings.

**Comment by AbdullahAlfaraj** [comment]
For those who don't want to move the folder back to the old location, you can create a symbolic link.
on Windows : 
Open cmd in the directory of the old location
```
mklink /D kohya_ss E:\kohya_ss
```
Where ```E:\kohya_ss``` is the new location. You could then launch kohya_ss from the symbolic link
